go.mod

```diff
@@ -86,3 +86,19 @@ replace (
 	k8s.io/metrics => k8s.io/kubernetes/staging/src/k8s.io/metrics v0.0.0-20190807140250-28e88e3f19c3
 	k8s.io/sample-apiserver => k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver v0.0.0-20190807140250-28e88e3f19c3
 )
+
+replace github.com/go-critic/go-critic v0.0.0-20181204210945-1df300866540 => github.com/go-critic/go-critic v0.3.5-0.20190526074819-1df300866540
```

Comment:
Sorry to bump you again, but can we check if this is still needed with [golangci-lint v1.18.0](https://github.com/golangci/golangci-lint/releases/tag/v1.18.0)?

Reply:
That works like a charm now. Thanks for checking!